node/src/Cardano/Node/API.hs

```diff
@@ -18,33 +19,97 @@ import           Servant
 import           Ntp.Client (NtpConfiguration, NtpStatus (..),
                      ntpClientSettings, withNtpClient)
 import           Ntp.Packet (NtpOffset)
-import           Pos.Chain.Block (LastKnownHeader, LastKnownHeaderTag)
-import           Pos.Chain.Update (UpdateConfiguration, curSoftwareVersion,
-                     withUpdateConfiguration)
+import           Pos.Chain.Block (HasBlockConfiguration, LastKnownHeader,
+                     LastKnownHeaderTag)
+import           Pos.Chain.Ssc (SscContext)
+import           Pos.Chain.Update (UpdateConfiguration, curSoftwareVersion)
 import           Pos.Client.CLI.NodeOptions (NodeApiArgs (..))
 import           Pos.Context
 import qualified Pos.Core as Core
+import           Pos.Core.JsonLog (CanJsonLog (..))
+import           Pos.Crypto (SecretKey)
 import qualified Pos.DB.Block as DB
 import qualified Pos.DB.BlockIndex as DB
 import qualified Pos.DB.Class as DB
 import           Pos.DB.GState.Lock (Priority (..), StateLock,
                      withStateLockNoMetrics)
-import qualified Pos.DB.Rocks.Functions as DB
-import qualified Pos.DB.Rocks.Types as DB
+import qualified Pos.DB.Rocks as DB
+import           Pos.DB.Txp (MempoolExt)
+import           Pos.DB.Txp.MemState (GenericTxpLocalData, TxpHolderTag)
 import           Pos.Infra.Diffusion.Subscription.Status (ssMap)
 import           Pos.Infra.Diffusion.Types
 import qualified Pos.Infra.Slotting.Util as Slotting
+import           Pos.Infra.Util.JsonLog.Events (jsonLogDefault)
+import           Pos.Launcher.Configuration (HasConfigurations)
 import           Pos.Launcher.Resource
 import           Pos.Node.API as Node
 import           Pos.Util (HasLens (..), HasLens')
 import           Pos.Util.CompileInfo (CompileTimeInfo, ctiGitRevision)
+import           Pos.Util.Lens (postfixLFields)
 import           Pos.Util.Servant
 import           Pos.Web (serveImpl)
+import qualified Pos.Web as Legacy
+import           Pos.WorkMode (RealModeContext)
+import           Pos.WorkMode.Class (WorkMode)
+
+type NodeV1Api
+    = "v1"
+    :> ( Node.API
+    :<|> Legacy.NodeApi
+    )
```

Comment:
Hooray, a `v1` prefix!

Reply:
As a matter of fact, I think we are now _"technically"_ at `v2` :joy:  ...